src/pytorch_lightning/utilities/device_parser.py

```diff
@@ -319,25 +44,49 @@ def parse_hpus(devices: Optional[Union[int, str, List[int]]]) -> Optional[int]:
     return int(devices) if isinstance(devices, str) else devices
 
 
-def num_cuda_devices() -> int:
-    """Returns the number of GPUs available.
-
-    Unlike :func:`torch.cuda.device_count`, this function will do its best not to create a CUDA context for fork
-    support, if the platform allows it.
-    """
-    if "fork" not in torch.multiprocessing.get_all_start_methods() or _is_forking_disabled():
-        return torch.cuda.device_count()
-    with multiprocessing.get_context("fork").Pool(1) as pool:
-        return pool.apply(torch.cuda.device_count)
+def determine_root_gpu_device(*args: Any, **kwargs: Any) -> Any:
+    rank_zero_deprecation(
+        "`pytorch_lightning.utilities.device_parser.determine_root_gpu_device` has been deprecated in v1.8.0 and will"
+        " be removed in v1.10.0. Please use `lightning_lite.utilities.device_parser.determine_root_gpu_device` instead."
+    )
+    return new_determine_root_gpu_device(*args, **kwargs)
 
 
 def is_cuda_available() -> bool:
-    """Returns a bool indicating if CUDA is currently available.
+    rank_zero_deprecation(
+        "`pytorch_lightning.utilities.device_parser.is_cuda_available` has been deprecated in v1.8.0 and will"
+        " be removed in v1.10.0. Please use `lightning_lite.utilities.device_parser.is_cuda_available` instead."
+    )
+    return new_is_cuda_available()
 
-    Unlike :func:`torch.cuda.is_available`, this function will do its best not to create a CUDA context for fork
-    support, if the platform allows it.
```

Comment:
git history for this file as well :)

Reply:
I know but I don't know how to do it. 

Discussed with @carmocca earlier this week. There will be many of these types of PRs where a part of the code needs to be kept and a part needs to be copied due to implementation differences. This will especially be true for the strategies, because short term (or maybe forever), many of them will have a PL-specifc implementation that is different from Lite. 

So I don't know how to resolve this. Suggestions welcome :)

Reply:
If it's a copy, then we cannot copy the git history. It would be only for moves

Reply:
yes but most of the times its the following pattern
- move file to lite
- create replacements with deprecation warnings in original place

If those two points are separate commits, the history for the moved file would be preserved